ortools_result.proto

```diff
@@ -11,6 +11,8 @@ message Activity {
   int32 alternative         = 5;
   int32 current_distance    = 6;
   string id                 = 7;
+  int64 lateness            = 8;
+  repeated float overloads  = 9;
```

Comment:
I am not sure about the `overloads` field because we cannot really say that "this" activity is the reason. Plus `overload`ing is a property of vehicle. This is like marking an activity "late" even though it does not violate its due date but just because the vehicle is violating its due date.